jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/LoggingMXBeanImpl.java

```diff
@@ -49,8 +57,60 @@
 /*[ENDIF]*/
 			PlatformLoggingMXBean {
 
-	private static final LoggingMXBeanImpl instance = new LoggingMXBeanImpl();
+	private static final LoggingMXBeanImpl instance;
+
+/*[IF Sidecar19-SE]
+	private static final Module java_logging;
+/*[ENDIF]*/
+	private static final Class<?> logManagerClass;
+	private static final Class<?> loggerClass;
+	private static final Class<?> levelClass;
```

Comment:
These three class references are only used in the static initializer: please make them local variables there.